### DISCUSSION TITLE
Optional Automatic Willpower

### DISCUSSION THREAD

**Discussion by Ralkiin** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#383] [Ideas]
An option to disable the automatic gain of willpower from pushed roll damage.

My group often pushes rolls for fun, as a joke, or to see "what would have happened if I had pushed that?" These rolls don't warrant the generation of willpower, and we have found that players quickly lose track of how much willpower they should have over the course of a session. I think this minor quality of life feature would be a good addition to the system, assuming it isn't a terribly difficult thing to code.

**Comment by aMediocreDad** [comment]
Perhaps something can be done with it. Unfortunately the side effect of automation. It's also a slippery slope in terms of feature flagging automation.
